omod/src/main/webapp/pages/manageAppointments.gsp

```diff
@@ -7,7 +7,7 @@
 <script type="text/javascript">
     <% if (breadcrumbOverride) { %>
         var breadcrumbs = _.flatten([
-        ${ breadcrumbOverride },
+        ${ ui.encodeJavaScript(breadcrumbOverride) },
```

Comment:
When I test this locally, the manageAppointment page loses most of the functionality it had before the patch. Are you getting similar behavior?